tests/func/test_step.py

```diff
@@ -33,8 +33,8 @@ def func():
         session.initialize(
             target=(run_as, code_to_debug),
             start_method=start_method,
-            env={'PTVSD_USE_CONTINUED': '1'},
             ignore_unobserved=[Event('continued')],
+            env={str('PTVSD_USE_CONTINUED'): str('1')},
```

Comment:
If this is always necessary on 2.7, `DebugSession` should take care of it on its end.